### PR TITLE
Update to run Next.js integration with Node.js v18

### PR DESCRIPTION
**Pull request by ijjk**
This allows configuring the Node.js version in chunk-tests so that we can update Next.js integration tests to run against Node.js v18 since it is now the min version allowed. 

x-ref: https://github.com/vercel/vercel/pull/10734#pullrequestreview-1688820489

### CODE REVIEW

**Review by styfle — DISMISSED** [review]
Thanks!

**Review by cb1kenobi — DISMISSED** [review]
LGTM

**Review by styfle — DISMISSED** [review]
Very nice, thanks!

File: .changeset/shaggy-bears-visit.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/next": patch
+---
+
+Update to run Next.js integration with Node.js v18
```
Comment (styfle) [inline .changeset/shaggy-bears-visit.md]:
```suggestion
---
---

```

This could be empty since its just tests

**Review by cb1kenobi — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Pull request merged**